src/frontend/screens/SyncModal/index.js

```diff
@@ -50,53 +49,20 @@ const SyncModal = ({ navigation }: Props) => {
     },
   ] = React.useContext(ConfigContext);
 
-  const [listen, setListening] = React.useState<boolean>(false);
-  const [peers, syncPeer, syncGetPeers] = usePeers(listen, deviceName);
-  const [ssid, setSsid] = React.useState<null | string>(null);
+  const { ssid } = useWifiStatus();
+  const [peers, syncPeer, syncGetPeers] = usePeers(!!ssid, deviceName);
```

Comment:
See https://github.com/digidem/mapeo-mobile/pull/637#discussion_r675168229 for context but wondering if using the `ssid` here makes sense, as opposed to having a separate `listen` variable that gets updated based on changes to the `ssid` via an effect

Reply:
addressed in https://github.com/digidem/mapeo-mobile/pull/637#discussion_r675639800